docs/forms.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_label: "Forms"
+sidebar_position: 4
+---
+
+# Forms
+
+After finishing setting up flutter and stacked (if you didn't setup stacked check out [get-started](get-started.md)), one of the features stacked provides is form handling by connecting form data in view and view model so that business logic and UI are separate and consistent with our goal. 
+
+## Form Setup
+
+Before generating methods for handling the controllers and validation messages. You need to tell the generator what your form has. And to tell the generator we use [annotations](https://medium.com/swlh/dart-annotations-a-simple-intro-to-reflection-c654275cc967). 
+
+Go into the view file and At the top of your class add
+
+```dart
+@FormView(fields: [])
+```
+
+`fields` accepts Lists of `FormField` which can be:
+
+- **FormTextField** 
+- **FormDateField** 
```

Comment:
Remove the DateField and Dropdown Field, we'll add separate sections for that when we have better examples and usage of it.